src/glyph/serialize.rs

```diff
@@ -52,7 +52,9 @@ impl Glyph {
         }
         let mut start = BytesStart::borrowed_name(b"glyph");
         start.push_attribute(("name", &*self.name));
-        start.push_attribute(("format", self.format.as_str()));
+        // we always serialize 2.0
+        //TODO: write out formatMinor if we start to support glif 2.1?
```

Comment:
Yes.

Reply:
I think this commend is outdated? We should indeed write out anything other than zero.

Reply:
We discard the version after loading (and error if it's something funny) so there's not much we can do here right now.